webui/react/src/utils/job.ts

```diff
@@ -5,6 +5,12 @@ import handleError, { DetError, DetErrorOptions, ErrorType } from 'utils/error';
 
 import { capitalize } from './string';
 
+// FIXME: what do I name this
+// This marks scheduler types that do not support fine-grain control of
+// job positions in the queue.
+export const unsupportedQPosition = new Set([
+  Api.V1SchedulerType.FAIRSHARE, Api.V1SchedulerType.KUBERNETES,
+]);
```

Comment:
**super nit:** space after line 13 to keep exported objects apart (unless they are very related - e.g. `jobTypeIconName` and `jobTypeIconUrl`)